paf-mvp-frontend/src/utils/queue.ts

```diff
@@ -0,0 +1,64 @@
+import { Log } from '@core/log';
+
+const log = new Log('PAF', '#3bb8c3');
+
+// API for loading PAF asynchronously and run functions regardless
+// whether it gets loaded before or after their script executes.
+//
+// <script src="url/to/PAF-lib.js" async></script>
+// <script>
+//  var PAF = PAF || {};
+//  PAF.queue = PAF.queue || [];
+//  PAF.queue.push(functionToExecuteOncePrebidLoads);
+// </script>
+//
+// In asynchronous code:
```

Comment:
I'm not sure "asynchronous code" is the relevant wording. In this code actually the commands will be ran synchronously, one after the other. "In the library" would be more appropriate, I think.

Reply:
Indeed. But the "In the library" isn't also a good wording because it is on the page in most cases. I reworded.